### PR TITLE
Added view config, minor cleanup

### PR DESCRIPTION
**Pull request by nyurik**

### CODE REVIEW

File: src/embed.js [hunk 1]
```diff
@@ -118,11 +118,11 @@ function embed(el, spec, opt, callback) {
 
   var runtime = vega.parse(spec, opt.config); // may throw an Error if parsing fails
   try {
-    var view = new vega.View(runtime)
+    var view = new vega.View(runtime, opt.viewConfig)
```
Comment (domoritz) [inline src/embed.js]: Hmm, why not just pass in `opt` and remove the calls to render below? 

Regardless, please add some documentation to the readme.

File: src/embed.js [hunk 2]
```diff
@@ -177,7 +175,10 @@ function embed(el, spec, opt, callback) {
           .text('Open in Vega Editor')
           .attr('href', '#')
           .on('click', function() {
-            post(window, embed.config.editor_url, {spec: source, mode: mode});
+            post(window, embed.config.editor_url, {
+              spec: JSON.stringify(spec, null, 2),
```
Comment (domoritz) [inline src/embed.js]: I think the editor should take care of formatting.

Comment (nyurik) [inline src/embed.js]: perhaps.  In the last patch, i simply rearranged the existing code to only do stringify when actually needed. Feel free to remove it outright :)

Comment (domoritz) [inline src/embed.js]: Oh, I missed that.

**Comment by domoritz** [comment]
Thank you for the patch!

**Pull request merged**